cli/tika_config_tester.php

```diff
@@ -0,0 +1,116 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * CLI config tester
+ *
+ * @package    search
+ * @copyright  2023 David Castro <david.castro@moodle.com>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+define('CLI_SCRIPT', true);
+
+require(__DIR__.'/../../../../config.php');
+require_once($CFG->libdir.'/clilib.php');      // Cli only functions.
+
+list($options, $unrecognized) = cli_get_params([
+    'help' => false,
+    'testfileid' => '',
+], [
+    'h' => 'help',
+    't' => 'testfileid',
+]);
+
+if ($unrecognized) {
+    $unrecognized = implode("\n  ", $unrecognized);
+    cli_error(get_string('cliunknowoption', 'admin', $unrecognized));
+}
+
+if ($options['help']) {
+    $help = "
+Run Tika diagnostics.
+
+Options:
+-h, --help              Print out this help
+-t, --testfileid        (Optional) PDF or accepted file id to send to tika for analysis
```

Comment:
Where do we get testfileid from? Is it id from mdl_files? If so, then this is not user friendly. Maybe we can pass a path to a test file instead? Or we could use files from  tests/fixtures?